lib/Fencer/Time.hs

```diff
@@ -26,8 +26,15 @@ newtype Timestamp = Timestamp Int64
     deriving newtype (Hashable, Enum)
 
 -- | Get current time as a 'Timestamp'.
+--
+-- The 'clock' library is used here because it has support for leap
```

Comment:
It's more like the other way around – `clock` does not support leap seconds 🙂 

I think the first sentence should simply be removed – it's bound to be confusing one way or the other.

Reply:
Ok, I fixed that.